### PR TITLE
Feature/sitemap

### PR DESCRIPTION
**Pull request by stentrop**

### CODE REVIEW

File: src/Extensions/BlogSitemapPattern.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        /** @var PluginSeoSitemapService $seoSitemapService */
+        $seoSitemapService = pluginApp(PluginSeoSitemapService::class);
+
+        /** @var AssistantsService $assistantsService */
+        $assistantsService = pluginApp(AssistantsService::class);
+        $dynamoPosts = $assistantsService->getDynamoDbPosts();
+        $result = [];
+        foreach($dynamoPosts as $post) {
+            $url = $blogService->buildFullPostUrl($post);
```
Comment (stentrop) [inline src/Extensions/BlogSitemapPattern.php:29]: Show only already active and published posts

File: src/Extensions/BlogSitemapPattern.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            $result[] = [
+                'publish_date' => date('Y-m-d', strtotime($post['data']['post']['publishedAt'])),
+                'url' => $url['postUrl'],
```
Comment (stentrop) [inline src/Extensions/BlogSitemapPattern.php:32]: Domain is missing

File: src/Extensions/BlogSitemapPattern.php (hunk 1 above)
Comment (stentrop) [reply src/Extensions/BlogSitemapPattern.php:29]: check if the post ist for the correct client

File: src/Extensions/BlogSitemapPattern.php (hunk 2 above)
Comment (ivladu-plenty) [reply src/Extensions/BlogSitemapPattern.php]: Updated as requested

File: src/Extensions/BlogSitemapPattern.php (hunk 1 above)
Comment (ivladu-plenty) [reply src/Extensions/BlogSitemapPattern.php]: The active and published posts have been filtered, the available posts are only for the current client so the second comment is not applicable

File: src/Extensions/BlogSitemapPattern.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        /** @var WebstoreConfigurationRepositoryContract $webstoreConfigurationRepository */
+        $webstoreConfigurationRepository = pluginApp(WebstoreConfigurationRepositoryContract::class);
+        $domain = $webstoreConfigurationRepository->getWebstoreConfiguration()->domainSsl;
+
+        /** @var AssistantsService $assistantsService */
+        $assistantsService = pluginApp(AssistantsService::class);
+        $dynamoPosts = $assistantsService->getDynamoDbPosts();
+        $result = [];
+        foreach($dynamoPosts as $post) {
+            if ($post['data']['post']['active'] === "true" && !is_null($post['data']['post']['publishedAtHour'])) {
```
Comment (stentrop) [inline src/Extensions/BlogSitemapPattern.php:34]: Future posts are still visible

Comment (rbodnariu-plenty) [reply src/Extensions/BlogSitemapPattern.php]: Updated as requested.

**Review by ivladu-plenty — APPROVED** [review]

**Pull request merged**